internal/driver/driver.go

```diff
@@ -147,9 +151,54 @@ func (d *Driver) Initialize(lc logger.LoggingClient, asyncCh chan<- *sdkModels.A
 
 	// Make sure the paths of existing devices are up-to-date.
 	go d.RefreshExistingDevicePaths()
-
+	go d.StartRtspCredentialServer()
 	return nil
 }
+func (d *Driver) StartRtspCredentialServer() {
+	d.lc.Infof("starting rtsp server\n")
+	rtspAuthServer := http.NewServeMux()
+	rtspAuthServer.HandleFunc("/rtspauth", d.RtspCredentialsHandler)
+	http.ListenAndServe("localhost:8000", rtspAuthServer)
```

Comment:
didn't you already have the option to make the port configurable? I think we need that.